paper/paper.md

```diff
@@ -0,0 +1,331 @@
+1. Perform version control using the Unix shell and git.
+1. Create reproducible reports using R markdown.
+1. Share code with others on GitHub.
+
+We believe these skills provide a solid foundation from which learners can teach
+themselves any additional coding skills that they will need in the future.
+
+## Course Content
+
+Our curriculum consists of modules that cover setup, data analysis and
+visualization, version control, sharing code, and writing reports.
+We also provide a goal for the workshop: to create a fictitious report for the
+United Nations on the relationship between gross domestic product (GDP), life
+expectancy, and CO~2~ emissions.
+The R programming lessons take a "tidyverse first" approach
+[@robinson_teach_2017] so learners can meet this goal.
+The modules are:
+
+0. Setup
+1. Welcome
+1. R for plotting (uses the tidyverse R packages)
+1. The Unix shell
+1. Git and GitHub
+1. R for data analysis (uses the tidyverse R packages)
+1. Writing reports in R markdown (uses the rmarkdown R package [@xie_r_2018])
+1. Group practice exercises
+1. Where to go from here
+
+Each lesson builds on the previous lessons.
+The Unix shell, git, and GitHub are introduced using the files generated in the
+R for plotting lesson.
+The lesson content for subsequent modules is then intermittently committed and
+pushed to GitHub.
+The written report combines all of the skills learned previously to write a
+report that one could share with the United Nations.
+Then, the learners have a chance to put everything they have learned into
+practice by working in small groups on practice problems that cover the course
+content.
+Finally, we end with a short module recapping what the workshop covered and how
+learners can continue to get help and learn more after the workshop ends.
+
+## Instructional Design
+<!-- teaching philosophy / pedagogy -->
+
+Our modules and instructor suggestions are developed in the style of [Software Carpentry](https://software-carpentry.org/):
+
+1. Each module contains learning objectives at the beginning of each lesson and
+a summary of key points.
+1. The five core modules (2 to 6) are designed to be taught via live coding of
+the content to learners.
+This is a central feature of Carpentries lessons, and we believe it is a great
+way to learn how to program because it requires learners to follow along and
+encounter errors that they must debug along the way, fostering additional
+questions about the course content.
+1. We incorporate formative assessments in the form of short practice exercises
+throughout each lesson so that learners can practice what they have learned and
+instructors can gauge learner understanding of the material.
+
+We also incorporate a few additional key components into the curriculum:
+
+1. Each lesson builds off of others, with the goal of creating a final report
+that can be shared with others.
+1. We structured the curriculum such that it could be taught through an
```

Comment:
Maybe worth adding a sentence to specify benefit of virtual workshops? e.g. virtual workshops was necessary during pandemic but are also useful to allow people from a variety of geographic locations to attend